src/components/NewWorkspaceModal.js

```diff
@@ -134,22 +134,22 @@ export default _.flow(
     })
 
     return Utils.cond(
-      [loading, spinnerOverlay],
+      [loading, () => spinnerOverlay],
       [hasBillingProjects, () => h(Modal, {
         title: Utils.cond(
-          [title, title],
-          [cloneWorkspace, 'Clone a workspace'],
-          [Utils.DEFAULT, 'Create a New Workspace']
+          [title, () => title],
+          [cloneWorkspace, () => 'Clone a workspace'],
+          [true, () => 'Create a New Workspace']
```

Comment:
Why use true instead of `Utils.DEFAULT`? I personally think the default is clearer on meaning.

Reply:
I saw below that there are also some that just have a default value after without a conditional. Maybe these should move to that?

Reply:
`Utils.DEFAULT` is a symbol with special meaning to `Utils.switchCase`. While it technically works here, I think it's misleading, because `cond` only looks for truthiness, so there's nothing special about `DEFAULT` as opposed to `true` or any other truthy value.

As for expressing the default, I've been pondering whether it would be good to also remove the special handling of the 'hanging function', and have everything use this form instead. Still thinking on that.